server/scripts/dbNode.ts

```diff
@@ -392,29 +392,116 @@ async function mainProcess(producer: RabbitMqHandler, pool: Pool) {
   }
 }
 
-// begin process
-log.info('db-node initialization');
+let pool, producer, numWorkers, workerNumber;
+async function initializer(): Promise<void> {
+  // begin process
+  log.info('db-node initialization');
+
+  // setup sql client pool
+  pool = new Pool({
+    connectionString: DATABASE_URI,
+    ssl: {
+      rejectUnauthorized: false,
+    },
+    max: 3,
+  });
+
+  pool.on('error', (err, client) => {
+    log.error('Unexpected error on idle client', err);
+  });
+
+  // get all dyno's list
+  const res = await fetch(
+    'https://api.heroku.com/apps/commonwealth-staging2/dynos',
```

Comment:
Should replace the `commonwealth-staging2` string with `HEROKU_APP_NAME` env var.